scripts/NGI_pipeline_test.py

```diff
@@ -0,0 +1,238 @@
+        for lane in (1,2,3,4,5,6,7,8):
+            # create fake sample names, start easy, one sample per lane
+            sample_name = create_sample_name(project_identifier, sample_number, args.facility)
+            # create the sample folder
+            sample_dir = os.path.join(FC_name, "Demultiplexing", project_name_underscore, "Sample_{}".format(sample_name))
+            os.mkdir(sample_dir)
+            sampinfo={ 'sampleid' : sample_name,
+                    'received' : datetime.datetime.today().strftime("%Y-%m-%d"),
+                    "total_autosomal_coverage" : "0",
+                    "libs":{}
+                    }
+            library = "A" # only a single library
+            sampinfo['libs'][library]={}
+            sampinfo['libs'][library]['libprepid']=library
+            sampinfo['libs'][library]['seqruns']={}
+            
+            sampinfo['libs'][library]['seqruns'][FC_name]={}
+            sampinfo['libs'][library]['seqruns'][FC_name]['seqrunid']=FC_name
+            sampinfo['libs'][library]['seqruns'][FC_name]['mean_autosomal_coverage']=0
+            sampinfo['libs'][library]['seqruns'][FC_name]['sequencing_status']={1:"DONE"}
+            sampinfo['libs'][library]['seqruns'][FC_name]['total_reads']=0
+            # store sampinfo
+            charon_data['samples'][sample_name]=sampinfo
+            # now copy the fastq file
+            fastq1_dest = os.path.join(sample_dir, "{}_S{}_L00{}_R1_001.fastq.gz".format(sample_name, lane, lane))
+            fastq2_dest = os.path.join(sample_dir, "{}_S{}_L00{}_R2_001.fastq.gz".format(sample_name, lane, lane))
+            if args.symlinks:
+                os.symlink(args.fastq1, fastq1_dest)
+                os.symlink(args.fastq2, fastq2_dest)
+            else:
+                shutil.copyfile(args.fastq1, fastq1_dest)
+                shutil.copyfile(args.fastq2, fastq2_dest)
+            
+            sample_number += 1
+    # now save everything to charon
+    print "updaiting charon-dev"
+    writeProjectData(charon_data, args.token, args.url)
+
+    print "now run this for each FC"
+    print "    python <path/to>/ngi_pipeline/scripts/ngi_pipeline_start.py organize <path/to/FC>"
+    print "and then this to start the analysis"
+    print "    python <path/to>/ngi_pipeline/scripts/ngi_pipeline_start.py analyze project<path/to>y/DATA/{}".format(project_identifier)
+
+
+
+
+def touch(file):
+    open(file, "w").close()
+    
+    
+def create_sample_name(project_identifier, sample_number, facility):
+    sample_id = str(sample_number).zfill(3)
+    if facility == 'stockholm':
+        return "{}_{}".format(project_identifier, sample_id)
+    else:
+        return "U{}".format(sample_id)
+    
+        
+
+
+
+def writeProjectData(data, token, base_url):
+    project=data
+    samples=project.pop('samples', None)
+    url=base_url+'/api/v1/project'
+    projson=json.dumps(project)
+    writeToCharon(projson, url, token)
```

Comment:
Yes right, but as I am creating the folder structure it feels better to create the corresponding structure in charon... I can use the `facility` option here and return in case of uppsala